### ISSUE TITLE
Visually display customizable search keyboard shortcut to user

### ISSUE THREAD

**Issue by vhscom** [docsifyjs/docsify#2391]
#### Problem or desire

#2278 added keyboard shortcuts for search and, I believe, collapsing the sidebar. In order to make it easier for users to discover keyboard shortcuts associated with search I'd like the ability to display a `kbd` key combo so users can quickly find it like:

<img width="279" alt="Screenshot 2024-03-27 at 11 25 41 AM" src="https://github.com/docsifyjs/docsify/assets/97140109/33b455c5-9ab9-4ce9-a540-ece641cb8820">

#### Proposal

Add and style a presentational keyboard shortcut key for sighted users. If there's more than one keyboard shortcut for search, allow the user to pick one keyboard option to display to the user.

#### Implementation

Something like:

```html
<kbd class="keyboard-shortcut"><span class="keyboard-shortcut-keys">⌘</span>K</kbd>
```

**Comment by jhildenbiddle** [comment]
Hi @vhscom.

A keyboard shortcut icon was added in #2278 (issue) / #2279 (PR). It currently shows a forward slash. You can review this on our `develop` branch [preview](https://docsify-preview.vercel.app/preview/). This feature will not be available publicly until Docsify v5 is released.

<img src="https://github.com/docsifyjs/docsify/assets/442527/ef0f3049-de8d-47d5-aa5c-f8f975f5cc17" height="40">

```html
<kbd title="Press / to search">/</kbd>
```

FYI, both Command+K and Control+K work as well as search shortcuts. I opted to display <kbd>/</kbd> because it is a single keystroke and universal across operating systems. Conditionally displaying <kbd>⌘ K</kbd> for macOS or <kbd>⌃ K</kbd> for other operating systems require OS detection which I was hoping to avoid. That said, I'm not opposed to making the switch since Command/Control+K is more common.

**Comment by jhildenbiddle** [comment]
This issue has been addressed by PR #2469. The changes will be available when Docsify v5 is released.

![CleanShot 2024-07-19 at 11 42 05@2x](https://github.com/user-attachments/assets/8b995eed-20c8-4b27-b63c-2265cca965f5)
